modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/liferay-clay.tld

```diff
@@ -739,6 +739,11 @@
 			<required>false</required>
 			<rtexprvalue>true</rtexprvalue>
 		</attribute>
+		<attribute>
+			<name>sorting</name>
```

Comment:
Would it make more sense to call this sortItems?

Reply:
Agree with you, but I wasn't sure if I had to be aligned with BE patterns or with the property already set in front-end side.
In the FE management of data-set-display it is called sorting.